stockflux-securities-master/src/components/AddSecurityButton.js

```diff
@@ -1,13 +1,15 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import "./AddSecurityButton.css";
+import Button from "./Button";
 
 const AddSecuritiesButton = ({ size }) => {
```

Comment:
Filename and component names do not match. I would suggest naming them both to `ButtonAddSecurity`.

Reply:
Also maybe it would be worth having folder named "buttons" with all 3 types of buttons you have.

Reply:
I've used your naming convention and it has the same name as the file now